console/src/components/UIKit/atoms/Spinner/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { SpinnerStyles } from './Spinner.style';
+
+const Spinner = props => {
+  const { size } = props;
+
+  // Spinner size (width & height ~ px values) depends upon the size prop received from parent <Button />
+
+  const spinnerWidth = size === 'small' ? 17 : 20;
+
+  const spinnerHeight = size === 'small' ? 17 : 20;
+
+  // ****************************** //
+
+  return (
+    <SpinnerStyles {...props} height={spinnerHeight} width={spinnerWidth}>
+      <div className={'sk_circle1' + ' ' + 'sk_child'} />
```

Comment:
If we introduce CSS-in-JS we should stick to it. So don't introduce class names in CSS, but use styled components or  styles created with `css` function.

Reply:
ok

Reply:
can we skip this in the Spinner case, I got some issues while passing a variable to rotate function?